### PR TITLE
faster-rcnn-resnet101-coco-sparse-60-0001

### PR DESCRIPTION
**Pull request by eldercrow**
Add FRCNN with 60% sparsity.

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: models/intel/faster-rcnn-resnet101-coco-sparse-60-0001/description/faster-rcnn-resnet101-coco-sparse-60-0001.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  where:
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width.
+  Expected color order is BGR.
+
+## Outputs
+
+1. The net outputs a blob with the shape: [300, 7], where each row is consisted of [`box_id`, `class_id`, `confidence`, `x0`, `y0`, `x1`, `y1`'], respectively.
```
Comment (eizamaliev) [inline models/intel/faster-rcnn-resnet101-coco-sparse-60-0001/description/faster-rcnn-resnet101-coco-sparse-60-0001.md]: Add description of `box_id` as well.

File: models/intel/faster-rcnn-resnet101-coco-sparse-60-0001/description/faster-rcnn-resnet101-coco-sparse-60-0001.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - `class_id`: ID of the detected class.
+    - `confidence`: [0, 1] detection score, the higher the value, the more confident the deteciton is on.
+    - `x0`, `y0`, `x1`, `y1`: normalized bounding box coordinates, range of [0, 1].
```
Comment (eizamaliev) [inline models/intel/faster-rcnn-resnet101-coco-sparse-60-0001/description/faster-rcnn-resnet101-coco-sparse-60-0001.md]: Please specify for which points these coordinates (e.g. bottom left corner)

File: models/intel/faster-rcnn-resnet101-coco-sparse-60-0001/description/faster-rcnn-resnet101-coco-sparse-60-0001.md (hunk 1 above)
Comment (vladimir-dudnik) [reply models/intel/faster-rcnn-resnet101-coco-sparse-60-0001/description/faster-rcnn-resnet101-coco-sparse-60-0001.md]: added description by analogy with public dense model

File: models/intel/faster-rcnn-resnet101-coco-sparse-60-0001/description/faster-rcnn-resnet101-coco-sparse-60-0001.md (hunk 2 above)
Comment (vladimir-dudnik) [reply models/intel/faster-rcnn-resnet101-coco-sparse-60-0001/description/faster-rcnn-resnet101-coco-sparse-60-0001.md]: added corners coordinates

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Pull request merged**